docs/tasks/debug-application-cluster/core-metrics-pipeline.md

```diff
@@ -0,0 +1,47 @@
+---
+approvers:
+- fgrzadkowski
+- piosz
+title: Core metrics pipeline
+---
+
+Resource usage metrics are important for:
+
+- Kubernetes components to operate (e.g. autoscalers to automatically scale user applications)
+- Kubernetes users to understand their applications behavior
+
```

Comment:
I think it's better to give more examples and less ambiguous phrases

```
Starting from Kubernetes 1.8, resource usage metrics, such as container CPU and memory usage,
are available in Kubernetes though the Metrics API. These metrics can be either accessed directly
by user, for example by using `kubectl top` command, or used by a controller in the cluster, e.g.
Horizontal Pod Autoscaler to make decisions.
```

Reply:
replaced